source/includes/gcp/_load_balancer.md

```diff
@@ -48,18 +48,101 @@ Retrieve a list of all load balancers in a given [environment](#administration-e
+`publicAddresses`<br/>*List<String>* | The list of external addresses
+`targetProxies`<br/>*List<String>* | The list of target proxies for this resource
 `urlMap`<br/>*String* | Server-defined URL for corresponding url map
 
+<!-------------------- RETRIEVE A LOAD BALANCER -------------------->
+
+#### Retrieve a load balancer
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/loadbalancers/8268558443601303519"
+
+# The above command returns JSON structured like this:
+```
+
+```json
+{
+    "data": {
+        "id": "8268558443601303519",
+        "forwardingRules": [
+            "https://www.googleapis.com/compute/v1/projects/test-area/global/forwardingRules/gfr-test-cme",
+            "https://www.googleapis.com/compute/v1/projects/test-area/global/forwardingRules/gfr-test-qxq"
+        ],
+        "targetProxies": [
+            "https://www.googleapis.com/compute/v1/projects/test-area/global/targetHttpProxies/tp-test-fkb",
+            "https://www.googleapis.com/compute/v1/projects/test-area/global/targetHttpsProxies/tp-test-rqi"
+        ],
+        "urlMap": "https://www.googleapis.com/compute/v1/projects/test-area/global/urlMaps/um-test-dfr",
+        "backendServices": [
+            "https://www.googleapis.com/compute/v1/projects/test-area/global/backendServices/be-test-jik"
+        ],
+        "name": "um-test-dfr",
+        "protocol": "HTTPS",
+        "publicAddresses": "34.98.108.48, 34.102.228.10"
+    }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/loadbalancers/:id</code>
+
+Retrieve a load balancer in a given [environment](#administration-environments)
+
+Attributes | &nbsp;
+------- | -----------
+`backendServices`<br/>*List<String>* | The list of backends for this resource
+`forwardingRules`<br/>*List<String>* | The list of forwarding rules for this resource
+`id`<br/>*String* | The id of this resource, which should be the same id as corresponding Url Map
+`name`<br/>*String* | The name of this resource
+`protocol`<br/>*String* | The protocol of this resource. If there are multiple target proxies, and at least one of them is HTTPS, the protocol for the load balancer is set to HTTPS. Valid values are HTTP or HTTPS.
+`publicAddresses`<br/>*List<String>* | The list of external addresses
+`targetProxies`<br/>*List<String>* | The list of target proxies for this resource
+`urlMap`<br/>*String* | Server-defined URL for corresponding url map
+
+<!-------------------- DELETE A LOAD BALANCER -------------------->
+
+#### Delete a load balancer
+
+```shell
+curl -X DELETE \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/loadbalancers/8268558443601303519"
+```
+
+```json
+{
+  "shortBackendsToDelete": "be-test-jik, be-other-srd",
```

Comment:
Shouldn't this be `"shortBackendsToDelete":[ "be-test-jik", "be-other-srd"]` ?

Reply:
<run/>

Reply:
Fixed